### PR TITLE
dialog title fix

### PR DESCRIPTION
**Pull request by benbaler**
closes https://github.com/hibobio/hibob/issues/24954

### CODE REVIEW

File: projects/ui-framework/src/lib/popups/dialog/dialog.component.scss [hunk 1]
```diff
@@ -33,6 +33,11 @@ $footerSize: 50px;
         div[b-dialog-sub-title] {
           margin-top: 5px;
         }
+        b-display-2 {
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/popups/dialog/dialog.component.scss]: not sure about this
who requested it?

Comment (benbaler) [reply projects/ui-framework/src/lib/popups/dialog/dialog.component.scss]: here: https://github.com/hibobio/hibob/issues/24954

**Comment by omrihecht** [comment]
@benbaler , i see,
the solution should be to break the word, not limit all titles to one line with ellipsis.

**Comment by galloween** [comment]
> @benbaler , i see,
> the solution should be to break the word, not limit all titles to one line with ellipsis.

@benbaler @omrihecht 

add
overflow-wrap: break-word;
to break long words. i used the same solution in tooltip for similar 'bug'

**Comment by benbaler** [comment]
@omrihecht @galloween done

File: projects/ui-framework/src/lib/popups/dialog/dialog.component.scss [hunk 2]
```diff
@@ -33,6 +33,11 @@ $footerSize: 50px;
...
+          overflow: hidden;
+          text-overflow: ellipsis;
+          overflow-wrap: break-word;
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/popups/dialog/dialog.component.scss]: remove:
```
overflow: hidden;
text-overflow: ellipsis;
```

**Pull request merged**